package.json

```diff
@@ -35,20 +35,20 @@
     "@types/web-bluetooth": "0.0.4"
   },
   "devDependencies": {
-    "browserify": "^15.0.0",
+    "browserify": "^15.2.0",
     "crc-32": "^1.0.2",
-    "del": "^3.0.0",
-    "gulp": "^3.9.1",
-    "gulp-sourcemaps": "^2.6.1",
-    "gulp-tslint": "^8.1.2",
-    "gulp-typescript": "^3.2.3",
-    "gulp-uglify": "^3.0.0",
+    "del": "^5.1.0",
+    "gulp": "^4.0.2",
+    "gulp-sourcemaps": "^2.6.5",
+    "gulp-tslint": "^8.1.4",
+    "gulp-typescript": "^5.0.1",
+    "gulp-uglify": "^3.0.2",
     "jszip": "^3.1.3",
     "merge2": "^1.2.0",
     "progress": "^2.0.0",
     "tslint": "5.8.0",
     "tslint-eslint-rules": "^4.1.1",
-    "typescript": "^2.6.2",
+    "typescript": "^3.8.3",
```

Comment:
Can you check this still compiles and works for older node versions, it currently supports version 6 and newer I believe